README.md

```diff
@@ -1,77 +1,89 @@
 
-### Optional Extras
+## Getting started
 
-In any order you like:
+```
+git clone https://github.com/amyj0rdan/bowling-challenge
+```
 
-* Create a nice interactive animated interface with jQuery.
-* Set up [Travis CI](https://travis-ci.org) to run your tests.
-* Add [ESLint](http://eslint.org/) to your codebase and make your code conform.
+## Usage
 
-You might even want to start with ESLint early on in your work — to help you
-learn Javascript conventions as you go along.
 
-## Bowling — how does it work?
+## Running tests
 
-### Strikes
+Open `SpecRunner.html` in your browser.
 
-The player has a strike if he knocks down all 10 pins with the first roll in a frame. The frame ends immediately (since there are no pins left for a second roll). The bonus for that frame is the number of pins knocked down by the next two rolls. That would be the next frame, unless the player rolls another strike.
+## Rules for bowling
 
-### Spares
+- There are 10 frames in a game of bowling.
+- There are 10 pins available to be knocked down.
+- Each frame consists of 2 throws, unless the 1st throw is a strike (ie, all pins are knocked down).
+- Score = number of pins knocked down + strike/spare bonus.
+- On the 10th frame, there can be a 3rd throw if the player scores a strike or spare on the first or second throw.
 
-The player has a spare if the knocks down all 10 pins with the two rolls of a frame. The bonus for that frame is the number of pins knocked down by the next roll (first roll of next frame).
+### Bonus scoring
 
-### 10th frame
+#### Strike
+Bonus for that frame is number of pins knocked down by the next two rolls.
 
-If the player rolls a strike or spare in the 10th frame they can roll the additional balls for the bonus. But they can never roll more than 3 balls in the 10th frame. The additional rolls only count for the bonus not for the regular frame count.
+#### Spare
+Bonus for that frame is the number of pins knocked down by the next roll.
 
-    10, 10, 10 in the 10th frame gives 30 points (10 points for the regular first strike and 20 points for the bonus).
-    1, 9, 10 in the 10th frame gives 20 points (10 points for the regular spare and 10 points for the bonus).
+#### 10th frame
 
-### Gutter Game
+Maximum of 3 balls can be rolled in the 10th frame. Player gets a bonus throw if they roll a strike or spare.
 
-A Gutter Game is when the player never hits a pin (20 zero scores).
+#### Gutter Game
 
-### Perfect Game
+Player never hits a pin (20 zero scores).
 
-A Perfect Game is when the player rolls 12 strikes (10 regular strikes and 2 strikes for the bonus in the 10th frame). The Perfect Game scores 300 points.
+#### Perfect Game
 
-In the image below you can find some score examples.
+Player rolls 12 strikes (10 regular strikes and 2 strikes for the bonus in the 10th frame). The Perfect Game scores 300 points.
 
-More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
+## Approach
```

Comment:
Super clear. Shows your approach really well and makes it obvious you had a logical process to implement features.